source/reference/minio-mc-admin/mc-admin-user-sts.rst

```diff
@@ -42,15 +42,15 @@ The :mc:`mc admin user sts` command has the following subcommands:
      - Description
 
    * - :mc-cmd:`mc admin user sts info`
-     - Retrieves information on the specified STS credential, including the parent user who generated the credentials and it's attached policies.
+     - Retrieves information on the specified STS credential, including the parent user who generated the credentials, attached policies, and expiration.
```

Comment:
I know `attached` is the correct term for policies, but perhaps the same `associated policies` phrasing below might work here too?

I suspect the `parent user` isn't the one attaching policies or doing something with the expiration.

Reply:
Matching what we say below makes sense!